From 21cbb9fb5fac3b3111ffac1de0f545a58c0c40e9 Mon Sep 17 00:00:00 2001
From: TobiasDev <tobiasdevde@gmail.com>
Date: Thu, 24 Dec 2020 23:37:29 +0100
Subject: [PATCH] fix resource packs


diff --git a/protocol/src/main/java/network/ycc/waterdog/pe/PERawPacketData.java b/protocol/src/main/java/network/ycc/waterdog/pe/PERawPacketData.java
index 88074783..2446fe9f 100644
--- a/protocol/src/main/java/network/ycc/waterdog/pe/PERawPacketData.java
+++ b/protocol/src/main/java/network/ycc/waterdog/pe/PERawPacketData.java
@@ -141,7 +141,7 @@ public class PERawPacketData {
         DefinedPacket.writeVarInt(pack.getSha256().length, resourcePackInfo);
         resourcePackInfo.writeBytes(pack.getSha256());
         resourcePackInfo.writeBoolean(false); //premium
-        resourcePackInfo.writeByte(1); //resource pack
+        resourcePackInfo.writeByte(6); //resource pack
         channel.write(resourcePackInfo);
     }
 
-- 
2.24.0.windows.2

